src/Plugins/Directory.Build.props

```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Neo\Neo.csproj" />
+    <ProjectReference Include="../../Neo/Neo.csproj" />
```

Comment:
I do not understand this `\` and `/` on dotnet.

Reply:
This change is not necessary for me,,,,, i normally use \

Reply:
Just making it consistent